File: BC_SENTDW-02/Sentencias/DTO/SentenciaDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PruebaBatch01.Sentencias.DTO
{
    class SentenciaDTO
    {
        private long id;
        private long beneficioTitular;
        private long cuilTitular;
        private long beneficioCausante;
        private long cuitCausante;
        private string expedienteAdministrativo;

        public SentenciaDTO()
        {
        }

        public long getId()
        {
            return id;
        }

        public void setId(long id)
        {
            this.id = id;
        }

        public long getBeneficioTitular()
        {
            return beneficioTitular;
        }

        public void setBeneficioTitular(long beneficioTitular)
        {
            this.beneficioTitular = beneficioTitular;
        }

        public long getCuilTitular()
        {
            return cuilTitular;
        }

        public void setCuilTitular(long cuilTitular)
        {
            this.cuilTitular = cuilTitular;
        }

        public long getBeneficioCausante()
        {
            return beneficioCausante;
        }

        public void setBeneficioCausante(long beneficioCausante)
        {
            this.beneficioCausante = beneficioCausante;
        }

        public long getCuitCausante()
        {
            return cuitCausante;
        }

        public void setCuitCausante(long cuitCausante)
        {
            this.cuitCausante = cuitCausante;
        }

        public string getExpedienteAdministrativo()
        {
            return expedienteAdministrativo;
        }

        public void setExpedienteAdministrativo(string expedienteAdministrativo)
        {
            this.expedienteAdministrativo = expedienteAdministrativo;
        }

        /*public SentenciaDTO mapToSentenciaDTO(Sentencias sentencia)
        {

          
            SentenciaDTO sentenciaDTO = new SentenciaDTO();
            sentenciaDTO.setId(sentencia.getId());
            sentenciaDTO.setBeneficioTitular(sentencia.getBeneficioTitular());
            sentenciaDTO.setCuilTitular(sentencia.getCuilTitular());
            if (sentencia.getBeneficioCausante() != null && sentencia.getBeneficioCausante() != null)
            {
                sentenciaDTO.setBeneficioCausante(sentencia.getBeneficioCausante());
                sentenciaDTO.setCuitCausante(sentencia.getCuilCausante());
            }
            sentenciaDTO.setExpedienteAdministrativo(sentencia.getExpedienteAdministrativo());
            return sentenciaDTO;
        }*/

    }
}

File: BC_SENTDW-01/src/main/java/ar/gov/anses/Sentencias/DTO/SentenciaDTO.java
package ar.gov.anses.Sentencias.DTO;


import ar.gov.anses.Sentencias.Domain.Sentencias;


public class SentenciaDTO {

    private Long id;
    private Long beneficioTitular;
    private Long cuilTitular;
    private long beneficioCausante;
    private Long cuitCausante;
    private String expedienteAdministrativo;

    public SentenciaDTO() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getBeneficioTitular() {
        return beneficioTitular;
    }

    public void setBeneficioTitular(Long beneficioTitular) {
        this.beneficioTitular = beneficioTitular;
    }

    public Long getCuilTitular() {
        return cuilTitular;
    }

    public void setCuilTitular(Long cuilTitular) {
        this.cuilTitular = cuilTitular;
    }

    public long getBeneficioCausante() {
        return beneficioCausante;
    }

    public void setBeneficioCausante(long beneficioCausante) {
        this.beneficioCausante = beneficioCausante;
    }

    public Long getCuitCausante() {
        return cuitCausante;
    }

    public void setCuitCausante(Long cuitCausante) {
        this.cuitCausante = cuitCausante;
    }

    public String getExpedienteAdministrativo() {
        return expedienteAdministrativo;
    }

    public void setExpedienteAdministrativo(String expedienteAdministrativo) {
        this.expedienteAdministrativo = expedienteAdministrativo;
    }

    public SentenciaDTO mapToSentenciaDTO(Sentencias sentencia){
        SentenciaDTO sentenciaDTO  = new SentenciaDTO();
        sentenciaDTO.setId(sentencia.getId());
        sentenciaDTO.setBeneficioTitular(sentencia.getBeneficioTitular());
        sentenciaDTO.setCuilTitular(sentencia.getCuilTitular());
        if(sentencia.getBeneficioCausante()!= null && sentencia.getBeneficioCausante()!=null){
            sentenciaDTO.setBeneficioCausante(sentencia.getBeneficioCausante());
            sentenciaDTO.setCuitCausante(sentencia.getCuilCausante());
        }
        sentenciaDTO.setExpedienteAdministrativo(sentencia.getExpedienteAdministrativo());
        return sentenciaDTO;
    }
}
